src/components/Collection/Series/EditSeriesTabs/GroupTab.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { mdiCheckUnderlineCircleOutline, mdiCloseCircleOutline, mdiMagnify, mdiPencilCircleOutline } from '@mdi/js';
+import cx from 'classnames';
+import { debounce } from 'lodash';
+
+import Input from '@/components/Input/Input';
+import { useLazyGetGroupInfinitesQuery } from '@/core/rtkQuery/splitV3Api/collectionApi';
+import { useGetSeriesGroupQuery } from '@/core/rtkQuery/splitV3Api/seriesApi';
+
+import type { CollectionGroupType } from '@/core/types/api/collection';
+
+type Props = {
+  seriesId: number;
+};
+
+function GroupTab({ seriesId }: Props) {
+  const [name, setName] = useState<string>('');
+  const [search, setSearch] = useState<string>('');
+  const [nameEditable, setNameEditable] = useState<boolean>(false);
+
+  const groupQuery = useGetSeriesGroupQuery({ seriesId: seriesId.toString(), topLevel: false }, {
+    refetchOnMountOrArgChange: false,
+  });
+
+  const [fetchGroup, groupResults] = useLazyGetGroupInfinitesQuery();
+
+  const getAniDbGroup = (): CollectionGroupType[] => {
+    const pages = groupResults.data?.pages;
+    if (!pages) return [];
+
+    const keys = Object.keys(pages);
+    if (!keys?.length) return [];
+
+    return pages[1];
+  };
+
+  const searchGroup = useMemo(() =>
+    debounce(async () => {
+      await fetchGroup({
+        startsWith: search?.length ? search : undefined,
```

Comment:
Condition not required. Check similar comment in `NameTab.tsx`
Add `if (!search) return;` to `useEffect` where this function is called.